django_perf_rec/orm.py

```diff
@@ -65,3 +66,24 @@ def patch_Q():
     def __init__(self, *args, **kwargs):
         super(Q, self).__init__(children=list(args) + sorted(kwargs.items()))
     Q.__init__ = __init__
+
+
+def patch_delete():
+    patchy.patch(get_candidate_relations_to_delete, """\
+        @@ -4,9 +4,12 @@def get_candidate_relations_to_delete(opts):
+            candidate_models = {opts}
+            candidate_models = candidate_models.union(opts.concrete_model._meta.proxied_children)
+            # For each model, get all candidate fields.
+        -   candidate_model_fields = set(chain.from_iterable(
+        -       opts.get_fields(include_hidden=True) for opts in candidate_models
+        -   ))
+        +   candidate_model_fields = []
+        +   for opts in candidate_models:
+        +       for field in opts.get_fields(include_hidden=True):
+        +           if field not in candidate_model_fields:
+        +               candidate_model_fields.append(field)
+        +
```

Comment:
it might be simpler to leave the code that creates the set and then add a line that turns it into a sorted list, rather than rewriting it?